syntax = "proto3";
package QikkDB.NetworkClient.Message;

message InfoMessage {
    enum StatusCode
    {
        OK = 0;
        WAIT = 1;
		GET_NEXT_RESULT = 6;
        QUERY_ERROR = 2;
        IMPORT_ERROR = 3;
        CONN_ESTABLISH = 4;
        CONN_END = 5;
		HEARTBEAT = 7;
    }
    StatusCode Code = 1;
    string Message = 2;
}